### ISSUE TITLE
[cloud-tensorflow-mnist]:

### ISSUE THREAD

**Issue by limorigu** [googlecodelabs/feedback#514]
Hello! Thanks for putting this great tutorial together. Just a quick comment: the current python files available for the lab parts do not actually match the lines described in the online tutorial. Look, for example, at the very first lab compared to mnist_1.0_softmax.py. Multiple lines are out of order, the learning rate fed into the gradient descent is 0.003 online v. 0.005 in the python file, and while the python file calls the minimization on the optimizer object on one line, online its broken into two lines. 

I personally feel comfortable enough with python to not be too bothered by these differences, but this is making the bar for newcomers unnecessarily high. No reason to add confusion to a topic that can be confusing enough to begin with :)

Once again, other than that, it's great so far. Thanks again for all the hard work and for making it available!

**Comment by martin-gorner** [comment]
Thank you for your feedback. The entire lab was revamped for modern Tensorflow and now uses Keras as its high-level API. And I believe the discrepancies you mention are not there anymore.
